src/apdu/messages/sign_transaction.c

```diff
@@ -25,6 +25,7 @@
 #include "../transaction/transaction.h"
 #include "../ui/main/idle_menu.h"
 #include "../../xrp/xrp_helpers.h"
+#include "lib_standard_app/crypto_helpers.h"
```

Comment:
If you add `INCLUDES_PATH += ${BOLOS_SDK}/lib_standard_app` I think you will be able to remove the relative path in this include and just use `#include "crypto_helpers.h"`

Reply:
Fixed